klam-ssh/v2/authorizedkeys_command.sh

```diff
@@ -1,5 +1,20 @@
 #!/bin/bash -x
 
+USER=$1
+SYSDFILE="/etc/systemd/system/docker.service.d/10-enable-namespaces.conf"
```

Comment:
This is good for now, but this file may be changing soon. If so, I'll make sure to update this repo